index.html

```diff
@@ -1,164 +1,236 @@
+          don’t hestiate to contact me.
+        </p>
         <button class="about-button" type="button">Get My Resume</button>
-    </div>
-    <div class="about-div">
-        <img class="about-icon" src="images/languages.svg" alt="">
+      </div>
+      <div class="about-div">
+        <img class="about-icon" src="images/languages.svg" alt="" />
         <h4>Languages</h4>
         <ul class="about-list">
-            <li>JavaScript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>CSS</li> 
+          <li>JavaScript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>CSS</li>
         </ul>
-    </div>
-    <div class="about-div">
-        <img class="about-icon" src="images/frameworks.svg" alt="">
+      </div>
+      <div class="about-div">
+        <img class="about-icon" src="images/frameworks.svg" alt="" />
         <h4>Frameworks</h4>
         <ul class="about-list">
-            <li>Bootstrap</li>
-            <li>Ruby on rails</li>
-            <li>RSpec</li>
-            <li>Capybara</li> 
-            <li>Selenium</li> 
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>Capybara</li>
+          <li>Selenium</li>
         </ul>
-    </div>
-    <div class="about-div">
-        <img class="about-icon" src="images/skills.svg" alt="">
+      </div>
+      <div class="about-div">
+        <img class="about-icon" src="images/skills.svg" alt="" />
         <h4>Skills</h4>
         <ul class="about-list">
-            <li>Codekit</li>
-            <li>GitHub</li>
-            <li>Codepen</li>
-            <li>Gitlab</li> 
-            <li>Terminal</li> 
+          <li>Codekit</li>
+          <li>GitHub</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
         </ul>
-    </div>
+      </div>
+      <footer>
+      <div class="form-section">
+        <div class="contact-text">
+          <p>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </p>
+        </div>
+        <form class="form-class" action="https://formspree.io/f/mlezrpqz" method="POST">
+          <label for="name">
+            <input
+              type="text"
+              id="name"
+              name="full-name"
+              placeholder="Full name"
+              minlength="3"
+              maxlength="30"
+              required
+            />
+          </label>
+          <label for="email">
+            <input
+              type="email"
+              id="email"
+              name="email"
+              placeholder="Email address"
+              required
+            />
+          </label>
+          <label for="mail-text">
+            <textarea id="mail-text" name="mail-text" maxlength="500" required>Enter text here</textarea>
+          </label>
+          <button class="about-button" type="submit">Get in touch</button>
+        </form>
```

Comment:
- [ ] [OPTIONAL] I think it would be nice to make `cursor pointer` on this button 👆🏻 
- [ ] The button  looks great, however, based on project requirements, the button's background color should change on hover and on click: 
![image](https://user-images.githubusercontent.com/77038610/144260188-92afc761-75e7-4884-8864-8019d7f3bf76.png)

Please kindly add these effects 🤗